torch/_subclasses/functional_tensor.py

```diff
@@ -64,6 +64,14 @@ class FunctionalTensor(torch.Tensor):
         torch.ops.prim.device.default,  # type: ignore[has-type]
     ]
 
+    # These are ops that claim to be functional, but actually are not
```

Comment:
Maybe call these "maybe-mutating / maybe-aliasing" ops

We should probably consider adding a tag for these tbh (probably ok not to in this PR)